src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -10,251 +10,4 @@
  * file that was distributed with this source code.
  */
 
-// Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2018-11-20 15:30 UTC
```

Comment:
please keep the above in this file

Reply:
This actually is not needed because md5 hash will be recalculated on `src/Symfony/Bridge/PhpUnit/simple-phpunit.php `

Reply:
right